.devcontainer/devcontainer.json

```diff
@@ -1,44 +1,47 @@
 {
     "name": "Open Space Toolkit Astrodynamics",
-    "mounts": [
-        "source=ostk_astrodynamics_vscode_extensions,target=/home/vscode/.vscode-server/extensions,type=volume"
-    ],
-    "runArgs": [
-        "--name",
-        "open-space-toolkit-astrodynamics-dev",
-        "--privileged"
-    ],
-    "containerEnv": {
-        "PACKAGE_DEPLOY_TOKEN": "${localEnv:PACKAGE_DEPLOY_TOKEN}"
-    },
+    "image": "openspacecollective/open-space-toolkit-astrodynamics-development-non-root:latest",
+    "initializeCommand": "make build-development-image-non-root && mkdir -p ${localWorkspaceFolder}/build",
     "workspaceMount": "source=${localWorkspaceFolder},target=/app,type=bind",
     "workspaceFolder": "/app",
+    "containerUser": "vscode",
     "updateRemoteUserUID": true,
-    "image": "openspacecollective/open-space-toolkit-astrodynamics-development:latest",
-    "initializeCommand": "make build-development-image",
+    "runArgs": [
+        "--name",
+        "open-space-toolkit-astrodynamics-dev-non-root"
+
+    ],
+    "mounts": [
+        "source=ostk_astrodynamics_vscode_extensions,target=/home/vscode/.vscode-server/extensions,type=volume",
+        "source=${localEnv:HOME}/.ssh,target=/home/vscode/.ssh,type=bind,readonly",
+        "source=${localEnv:HOME}/.gitconfig,target=/home/vscode/.gitconfig,type=bind,readonly"
+    ],
     "shutdownAction": "stopContainer",
     "customizations": {
         "vscode": {
+            "extensions": [
+                "ms-vscode.cpptools",
+                "ms-python.python",
+                "ms-python.black-formatter",
+                "mhutchie.git-graph",
+                "ms-vscode.cpptools-extension-pack",
+                "mechatroner.rainbow-csv",
```

Comment:
Did you mean to add this csv extension? Was this something that slipped in from your local config?

Reply:
Yes I did, its quite useful for visualizing CSVs because it colors each column differently, and since there are plenty of validation files in the .csv format I figure its a nice tool to have.